src/Mantasflowers.Contracts/User/Request/UpdateUserAddressRequest.cs

```diff
@@ -0,0 +1,13 @@
+namespace Mantasflowers.Contracts.User.Request
+{
+    public class UpdateUserAddressRequest
+    {
+        public string Country { get; set; }
+
+        public string City { get; set; }
+
+        public string Street { get; set; }
+
+        public string Zipcode { get; set; }
+    }
+}
```

Comment:
Replace with:
```cs
public class UpdateUserAddressRequest : AddressTemplate { }
```
Can be already found in Common\Templates. Same with UpdateUserContactInfoRequest.

Reply:
Noticed that namings of request and response classes are missing Post/Get prefixes.

Reply:
Those prefixes were never supposed to be there in the first place. Just that some requests started with `Get` :D